Slam/AddMatchViewController.swift

```diff
@@ -11,7 +11,8 @@ public class AddMatchViewController: UIViewController, AddMatchView {
     override public func viewDidLoad() {
         super.viewDidLoad()
         interactor = DefaultAddMatchInteractor(
-            view: self
+            view: self,
+            api: SlamAPI(httpClient: AsynchronousHTTPClient())
```

Comment:
It looks like you made this change to `AddMatchViewController` without updating it's specs.